components/CreateOrUpdateRecipeForm.js

```diff
@@ -34,22 +67,35 @@ export default function CreateOrUpdateRecipeForm({
           $titleInput={inputValidation}
           defaultValue={recipeDetails.name}
         />
-
         <label htmlFor="duration">Cooking duration:</label>
         <input
           type="text"
           id="duration"
           name="duration"
           defaultValue={recipeDetails.preparationTime}
         />
-
-        <label htmlFor="imgurl">Image Url</label>
-
+        <label htmlFor="file">Image</label>
+        {hasImage ?? imagePreviewSrc ? (
+          <StyledImageContainer>
+            <div>
+              <StyledImage
+                src={imagePreviewSrc.src || recipeDetails.imageURL}
+                alt={recipeDetails.name || "new Image"}
+                width={300}
+                height={300}
+              />
+              <StyledTrasBin onClick={handleImageDelete} />
+            </div>
+          </StyledImageContainer>
+        ) : null}
```

Comment:
I think we can skip the ternary here, seeing as we only want to do something is `hasImage` and `imagePreviewSrc are truthy. `

My slight concern is that `imagePreviewSrc` is an object. That means it will always be truthy. Maybe you want to check `imagePreviewSrc.src`?

Or change the data type for `imagePreviewSrc` to be a string. This might require refactoring elsewhere. 
 

```suggestion
        {hasImage ?? imagePreviewSrc && (
          <StyledImageContainer>
            <div>
              <StyledImage
                src={imagePreviewSrc.src || recipeDetails.imageURL}
                alt={recipeDetails.name || "new Image"}
                width={300}
                height={300}
              />
              <StyledTrasBin onClick={handleImageDelete} />
            </div>
          </StyledImageContainer>
        )}
```